src/CommandLineUtils/HelpText/DescriptionFormatter.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Linq;
+using System.Text;
+
+namespace McMaster.Extensions.CommandLineUtils.HelpText
+{
+    /// <summary>
+    /// A formatter for creating nicely wrapped descriptions for display on the command line in the second column
+    /// of generated help text.
+    /// </summary>
+    public class DescriptionFormatter
```

Comment:
This class is coupled to the needs to DefaultHelpTextGenerator. I think the functionality in this class could be made generally useful with a few tweaks. Here are some suggestions for how this code could be changed to make a generally useful class that would work for DefaultHelpTextGenerator and others.

1. Consider naming it `HangingIndentWriter`.
2. Consider making the usage look like this. This would mean moving GetDefaultConsoleWidth back into DefaultHelpTextGenerator, and doing the addition of `firstColumnWidth + spacerLength` inside DefaultHelpTextGenerator instead passing them as separate constructor arguments.
```
var writer = new HangingIndentWriter(indentSize: 15, maxLineLength: 80);
writer.Write("blah blah");
```

Reply:
Excellent suggestion!